phoenix-core/src/it/java/org/apache/phoenix/end2end/DerivedTableIT.java

```diff
@@ -241,7 +241,7 @@ public void testDerivedTableWithWhere() throws Exception {
             rs = statement.executeQuery();
             assertFalse(rs.next());
 
-            // (offset) where
+            // (where offset)
```

Comment:
Guess it doesn't make too much sense here to keep this (where offset) case. You can simply remove it, or otherwise, make it "(where) offset" which is essentially the same as (where offset) but is more meaningful for testing query flattening.